.github/mergeable.yml

```diff
@@ -58,5 +58,5 @@ mergeable:
               regex: 'feature|bug|improvement|document|chore'
               message: 'Label must include one of the following: `feature`, `bug`, `improvement`, `document`, `chore`'
           - must_include:
-              regex: '3\.0\.x|3\.1\.x|3\.2\.0'
-              message: 'Label must include one or more of the following: `3.0.x`, `3.1.x`, `3.2.0`'
+              regex: '3\.0\.x|3\.1\.x|3\.2\.0|4\.0\.0'
```

Comment:
Let's change these hardcoded versions to something like `\d+.\d+.\d+`

Reply:
I'll fix it.